app/renderer/js/utils/request-util.ts

```diff
@@ -1,19 +1,19 @@
-const fs = require('fs');
-const Logger = require('./logger-util');
+import * as fs from 'fs';
+import { Logger } from './logger-util';
 
-const CertificateUtil = require(__dirname + '/certificate-util.js');
-const ProxyUtil = require(__dirname + '/proxy-util.js');
-const ConfigUtil = require(__dirname + '/config-util.js');
-const SystemUtil = require(__dirname + '/../utils/system-util.js');
+import { CertificateUtil } from './certificate-util';
+import { ProxyUtil } from './proxy-util';
+import { ConfigUtil } from './config-util';
+import { SystemUtil } from './system-util';
 
 const logger = new Logger({
 	file: `request-util.log`,
 	timestamp: true
 });
 
-let instance = null;
+let instance: any = null;
 
-class RequestUtil {
+export class requestUtil {
```

Comment:
@vsvipul why did you rename the class? It should be `RequestUtil`

Reply:
We needed to export an instance of RequestUtil to make it work in other files, otherwise tsc was returning an error. So renamed the class and named the instance RequestUtil.

Reply:
We can do that as `export { requestUtil as RequestUtil };`

Reply:
oh, will change. Thanks.